subhub/hub/stripe/controller.py

```diff
@@ -47,21 +47,21 @@ def view() -> tuple:
         logger.info("payload type", type=type(payload))
         sig_header = request.headers["Stripe-Signature"]
         event = stripe.Webhook.construct_event(payload, sig_header, CFG.HUB_API_KEY)
-        pipeline = StripeHubEventPipeline(event)
-        pipeline.run()
+        return {"message": ""}, 200
     except ValueError as e:
         # Invalid payload
         logger.error("ValueError", error=e)
-        return Response(status=400)
+        return {"message": e}, 400
```

Comment:
Do we need to update the Swagger for this?  https://github.com/mozilla/subhub/blob/22a4e08c1a5c59d9ddeab7b724e83d5df502c3ba/subhub/swagger.yaml#L498